### PR TITLE
APPRL-1   Readme file updated with some basic information.

### PR DESCRIPTION
**Pull request by MarieDale3**
Starter readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+The repository can be found here: https://github.com/SkillsFundingAgency/das-apprentice-learning
+
+# Database
+
+# Tests
+
+# Authors
+- Ben Stone
+- Marie Dale
+- Dan Beavon
```
Comment (DannyGB) [inline README.md]: These could include links to the respective github profiles

**Review by DannyGB — CHANGES_REQUESTED** [review]
I was expecting it to look more like the [template I put in the Jira ticket](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2).

File: README.md (hunk 1 above)
Comment (MarieDale3) [reply README.md]: Tried again, some sections left blank for now but the template is fully.  I can update the contributing section once created

File: README.md [hunk 2]
```diff
@@ -1,18 +1,38 @@
...
-# das-apprentice-learning
+# DAS Capture The Flag
 Capture the Flag Game is a .NET Core 3.0 web application.  It is a multi-player browser game.
 Once two user's have connected and selected start the game will begin.
 
 The repository can be found here: https://github.com/SkillsFundingAgency/das-apprentice-learning
 
-# Database
+## Prerequisites
+
```
Comment (DannyGB) [inline README.md:12]: You could fill this out, you have the information available to you, I can think of one pre-req off of the top of my head. If you're struggling to understand what goes here then have a look at some other projects readme's for some inspiration

**Review by DannyGB — CHANGES_REQUESTED** [review]
Great, adds some pre-reqs and we can merge

File: README.md [hunk 3]
```diff
@@ -18,8 +21,8 @@ The repository can be found here: https://github.com/SkillsFundingAgency/das-app
 
 
 ## Built With
-Framework:   .NET Core 3.0
-Architecture:   MVVM
+1.   Framework .NET Core 3.1 
+2.   Architecture MVVM
```
Comment (DannyGB) [inline README.md]: It's an MVC architecture

**Review by DannyGB — APPROVED** [review]

**Pull request merged**